content/authors/Christoph-Vogelbacher /_index.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
@Remi-Gau unwanted whitespace in folder name.

Reply:
Checked with Christophe: temporarily moved his content on a separate branch ready to be added (don't want to put his name there and make him feel he is now forced to do something).